server/test/Makefile

```diff
@@ -1,19 +1,30 @@
 CC = gcc
-CFLAGS = -std=c99 -Wall -pedantic -g
```

Comment:
Why are you removing `-Wall` and `-pendantic`? If the code doesn't compile with them, fix the code instead of removing the flags.